controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -67,6 +67,12 @@ const std::string wfa_ca::wfa_ca_status_to_string(eWfaCaStatus status)
     return std::string();
 }
 
+static bool is_hex_notation(std::string const &s)
```

Comment:
I would remove the "is" prefix.
Also, I think te convention in our code is to put the const before the type (nitpick)

Reply:
Agree on the const, disagree about the is